src/components/App/index.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
+import Game from '../Game'
 
-const App = () => (
-  <p>Helloo </p>
-);
+const App = () =>{
+  return (
+      <div>
```

Comment:
You don't need a div here as you're just returning Game, you can condense your code to: 

`export default App = () => <Game />`

Usually we follow the patterns of export default as a separate line but in this case you can just merge everything into the same line.